conan/cli/commands/inspect.py

```diff
@@ -24,20 +23,27 @@ def inspect(conan_api, parser, *args):
     Inspect a conanfile.py to return its public fields.
     """
     parser.add_argument("path", help="Path to a folder containing a recipe (conanfile.py)")
+    parser.add_argument("-r", "--remote", default=None, action="append",
+                        help="Remote names. Accepts wildcards ('*' means all the remotes available)")
+    parser.add_argument("-l", "--lockfile", action=OnceArgument,
+                        help="Path to a lockfile. Use --lockfile=\"\" to avoid automatic use of "
+                             "existing 'conan.lock' file")
 
     args = parser.parse_args(*args)
-
     path = conan_api.local.get_conanfile_path(args.path, os.getcwd(), py=True)
-
-    conanfile = conan_api.graph.load_conanfile_class(path)
-    ret = {}
-
-    for name, value in python_inspect.getmembers(conanfile):
-        if name.startswith('_') or python_inspect.ismethod(value) \
-           or python_inspect.isfunction(value) or isinstance(value, property):
-            continue
-        ret[name] = value
-        if value is None:
-            continue
-
-    return ret
+    lockfile = conan_api.lockfile.get_lockfile(lockfile=args.lockfile,
+                                               conanfile_path=path,
+                                               cwd=os.getcwd(),
+                                               partial=None)
+    remotes = conan_api.remotes.list(args.remote) if args.remote else []
+    conanfile = conan_api.local.inspect(path, remotes=remotes, lockfile=lockfile)
+    result = conanfile.serialize()
+    possible_options = conanfile.options.possible_values
```

Comment:
I think the options possible values might be inside ``conanfile.serialize()`` (it will exist also in ``conan graph info``, but maybe it is ok there too?)

Also, it might be better to have this as ``possible_options`` or ``options_definition`` or something like that, so the ``options`` member always has the same meaning (actual values)?